### PR TITLE
Travis kitchen tests

### PR DESCRIPTION
**Pull request by edwlarkey**

### CODE REVIEW

**Comment by atheiman** [comment]
To get around those gem install issues, first thing id try is updating the version of rubygems. You could do that in the travis script with `/opt/chef/embedded/bin/gem update --system`.  Im not sure what adverse effects that would have, but another option to try is not use latest version of chef (try `12.12.15` or something).

File: .travis.yml [hunk 1]
```diff
@@ -5,16 +5,24 @@ sudo: required
...
 
 script:
-- bundle exec ./.travis.sh
+  - bundle exec ./.travis.sh
 
 matrix:
   allow_failures:
-  - env: SUITE=lint
+    - env: SUITE=lint
+    - env: SUITE=ubuntu-1604
```
Comment (atheiman) [inline .travis.yml]: just add a note for why this is failing, and we're good to go I think

File: .kitchen.docker.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+provisioner:
+  name: dokken
+
+platforms:
+  - name: centos-6.8
+    driver:
+      image: centos:6.8
+      pid_one_command: /sbin/init
+      intermediate_instructions:
+        - RUN /usr/bin/yum install -y sudo
```
Comment (edwlarkey) [inline .kitchen.docker.yml]: We solved this by using a specified version of chef. 12.15.19 as you also suggested.

**Comment by atheiman** [comment]
👍 :1st_place_medal:

**Comment by atheiman** [comment]
@ARentz07 @ChiefAlexander think you can take a look at this and get it merged?

**Comment by ARentz07** [comment]
lgtm, 👍

**Pull request merged**